pitch.md

```diff
@@ -23,38 +37,125 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+Like LaTeX the programmer would write text to a file. 
+The programmer types out what would be on each line of the tableau tree,
+using special characters for logic symbols.
+The programmer should also have a way to draw a check mark at the end of each line of the tree.
+Tableau trees are binary trees so the programmer can also specify if the tree branches off.
+
+One possible syntax is
+```
+(A \or B) \and C \check
+C
+A \or B
+  A
+    branch1
+    branch2
+  B
+```
+to draw
+
+![tree](/images/tree.png)
+
+which is similar to how qtree, a LaTeX package for trees, structures code but this doesn't require entering math mode using \$.
+Anthony mentioned in his critique that the backslash may not be necessary in this language.
+This would restrict programmers from having variable names like 'or' and 'and', but then again those would be bad variable names in a logic proof.
 
 ## Operation (semantics)
 _What might happen when a program runs? Is there a computational model that
 corresponds with your domain? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
 might they be communicated to the user?_
 
+The file is read an compiled.
+A program would render a tree based on the text file and output an image file.
+This fits the data-entry model.
+Drawing a tree is a sub-problem of drawing a graph, which was the given example for the data-entry model.
+
+Syntax errors, such as branching in 3 directions, should be reported after compilation, before the tree is drawn.
+This could be reported by writing to an error stream or file.
+
+Semantic errors would result in a malformed tree.
+I'm not sure how this would be reported to the user other than by having an image file of a weird tree.
 
 ## Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to draw any binary tree with logic symbols.
+
+It should be possible, but maybe not necessarily easy, to draw trees with text that contains symbols other than logic symbols.
+Maybe Unicode symbols.
+It should be possible to use different kinds of logic symbols.
+For example, not can be represented with ¬ or !, the user should be able to choose which set of symbols they want to use.
+
```

Comment:
How would you communicate to the user that drawing a non-binary tree is not the correct way to use this DSL?